### PR TITLE
Remove last less file

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,13 +46,10 @@
     "unplugin-fonts": "^1.1.1",
     "vite": "4.5.0",
     "vite-plugin-mkcert": "^1.17.5",
-    "vite-plugin-pwa": "^0.20.0"
+    "vite-plugin-pwa": "^0.20.0",
+    "vite-plugin-env-compatible": "^2.0.1"
   },
   "scripts": {
-    "lint-dark": "sed '/^[\\.\\}\\@]/d;/^[[:blank:]]*\\..*/d;/\\/\\/.*/d' common/src/main/webapp/less/variables-dark.less > vars.css;trap \"rm vars.css\" EXIT;npx stylelint common/src/main/webapp/sass",
```
Comment (hugo-vrijswijk) [inline package.json:54]: Is the `sed` trick here no longer necessary? Because the script still seems useful to have

**Review by hugo-vrijswijk — COMMENTED** [review]
LGTM other than 1 question 🎉

File: package.json (hunk 1 above)
Comment (cquiroz) [reply package.json:54]: the variables file no longer exists. I could keep one for reference

Comment (hugo-vrijswijk) [reply package.json:54]: The `vars.css` is no longer needed? The CI check should probably also be updated then. But I meant the `stylelint` call

Comment (cquiroz) [reply package.json:54]: The current check is
`npx stylelint --formatter github common/src/main/webapp/sass`
It checks whatever is in that dir

Comment (hugo-vrijswijk) [reply package.json:54]: Yes but the script can still be useful to run locally, or to call it with `--fix` without having to remember what the exact command is

Comment (cquiroz) [reply package.json:54]: ok I'll restore them

**Pull request merged**